crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Effect.cs

```diff
@@ -245,6 +243,29 @@ internal void ExecuteEffect(CommandBuffer buffer, Material material, MaterialPro
             }
         }
 
+        static void UpdateGlobals(Material oceanMaterial)
+        {
+            {
+                // Unity is not setting the sun correctly both in scene view and before transparent pass. Use most
+                // likely sun candidate.
+                Shader.SetGlobalVector("_CrestWorldSpaceLightPos0", -RenderSettings.sun.transform.forward);
```

Comment:
If this is run each frame it should probably use param IDs rather than strings

Reply:
Done. I've added them using an approach I borrowed from Unity by placing them in a static class called ShaderIDs. LMKWYT.